HW2/HW2EX1/ses-retest/cmd.ap.sub_03

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env tcsh
+
+# created by uber_subject.py: version 0.40 (March 30, 2017)
+# creation date: Thu Aug  3 22:53:54 2017
+
+# set data directories
+set top_dir   =                                                          \
+    /Users/timmichaels/Documents/UConn/BIRC/WedJuly26/Non-Linear/playground/HW2/ses-retest
+set anat_dir  = $top_dir/anat
+set epi_dir   = $top_dir/func
+
+# set subject and group identifiers
+set subj      = sub_03
+set group_id  = retest
+
+# run afni_proc.py to create a single subject processing script
+afni_proc.py -subj_id $subj                                              \
+        -script proc.$subj -scr_overwrite                                \
+        -blocks tshift align tlrc volreg blur mask scale regress         \
+        -copy_anat $anat_dir/sub-03_ses-retest_T1w.nii.gz                \
+        -tcat_remove_first_trs 0                                         \
+        -dsets $epi_dir/sub-03_ses-retest_task-linebisection_bold.nii.gz \
+        -tlrc_base MNI_avg152T1+tlrc                                     \
+        -volreg_align_to third                                           \
+        -volreg_align_e2a                                                \
+        -volreg_tlrc_warp                                                \
+        -blur_size 6.0                                                   \
+        -regress_stim_times                                              \
```

Comment:
- You shouldn't combine regressors like this. The correct and incorrect response trials might be associated with different activations, but here the same activation parameters will be fit to both event types. 
- You also need the other trial types in here (the ones that don't go into the contrast)
- There's no contrast here. You need to include something like 
```-gltsym 'SYM: landmark -detection' -glt_label 1 landmark-detection
```